libcodechecker/libhandlers/store.py

```diff
@@ -22,7 +22,7 @@
 import zipfile
 import zlib
 
-from Authentication.ttypes import Permission
+from shared.ttypes import Permission
```

Comment:
don't we want to separate handlers by directory too?
like libhandlers/v6/store.py etc.

Reply:
`libhandlers` is the **subcommand handler**, it should NOT be versioned.

A `api_handlers` package in `server` could be introduced, but this can be done later on, even as early as version 6.1 or as late as version 7, as it is a purely internal thing, not breaking the client-server nor the user-client API.